check if we can get rid of the fsGroup config in the Helm chart

You currently have to set the fsGroup to 0 when running the Helm chart on K8s. People tend to forget that. However, we don't need to this for the subcharts. So I guess there is a way around it. We should check how our subcharts are handling this and then handle it the same way.

However, it doesn't work when I don't set it explicitly. 
In this case, `curl` to upload a model gives 500

```
[2020-01-30 16:47:47 +0000] [24] [ERROR] Exception occurred while handling uri: 'http://35.187.75.126/api/projects/default/models?api_token=b22f8904f9f55969db5ce8ec297f2bf458e61c38'
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/aiohttp/connector.py", line 924, in _wrap_create_connection
    await self._loop.create_connection(*args, **kwargs))
  File "uvloop/loop.pyx", line 1974, in create_connection
  File "uvloop/loop.pyx", line 1951, in uvloop.loop.Loop.create_connection
ConnectionRefusedError: [Errno 111] Connection refused

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/sanic/app.py", line 942, in handle_request
    response = await response
  File "/usr/local/lib/python3.6/site-packages/rasax/community/api/decorators.py", line 177, in decorated_function
    return await await_and_return_response(args, kwargs, request)
  File "/usr/local/lib/python3.6/site-packages/rasax/community/api/decorators.py", line 107, in await_and_return_response
    response = await response
  File "/usr/local/lib/python3.6/site-packages/rasax/community/api/blueprints/models.py", line 57, in upload_model
    minimum_version = await model_service.minimum_compatible_version()
  File "/usr/local/lib/python3.6/site-packages/rasax/community/services/model_service.py", line 120, in minimum_compatible_version
    info = await stack_service.status()
  File "/usr/local/lib/python3.6/site-packages/rasax/community/services/stack_service.py", line 53, in status
    return await self.stack_endpoint.request(subpath="/version", method="get")
  File "/usr/local/lib/python3.6/site-packages/rasa/utils/endpoints.py", line 146, in request
    **kwargs,
  File "/usr/local/lib/python3.6/site-packages/aiohttp/client.py", line 1005, in __aenter__
    self._resp = await self._coro
  File "/usr/local/lib/python3.6/site-packages/aiohttp/client.py", line 476, in _request
    timeout=real_timeout
  File "/usr/local/lib/python3.6/site-packages/aiohttp/connector.py", line 522, in connect
    proto = await self._create_connection(req, traces, timeout)
  File "/usr/local/lib/python3.6/site-packages/aiohttp/connector.py", line 854, in _create_connection
    req, traces, timeout)
  File "/usr/local/lib/python3.6/site-packages/aiohttp/connector.py", line 992, in _create_direct_connection
    raise last_exc
  File "/usr/local/lib/python3.6/site-packages/aiohttp/connector.py", line 974, in _create_direct_connection
    req=req, client_error=client_error)
  File "/usr/local/lib/python3.6/site-packages/aiohttp/connector.py", line 931, in _wrap_create_connection
    raise client_error(req.connection_key, exc) from exc
aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host chart-1580402718-rasa-x-rasa-production:5005 ssl:None [Connection refused]```

--- a/README.md
+++ b/README.md
@@ -49,7 +49,9 @@ recommend to set at least these values:
 | rabbitmq.rabbitmq.password           | Password for RabbitMq.                                                                     | `test`             |
 | global.postgresql.postgresqlPassword | Password for the Postgresql database.                                                      | `password`         |
 | global.redis.password                | Password for redis.                                                                        | `password`         |
-| securityContext.fsGroup              | fsGroup for the volume mounts. Should be `0` in case the  chart is deployed on Kubernetes. | `""`               |
+| securityContext.runAsUser            | The UID to run the entrypoint of container processes                                       | `1000`             |
+| securityContext.fsGroup              | A special supplemental group that applies to all containers in a pod                       | `1000`             |
+| securityContext.runAsNonRoot         | Indicates that the container must run as a non-root user                                   | `true`             |
 | Chart.appVersion                     | Version of Rasa X which you want to use.                                                   | `0.25.0`           |
 | rasa.tag                             | Version of Rasa OSS which you want to use.                                                 | `1.7.0`            |
 | app.name                             | Name of your action server image.                                                          | `rasa/rasa-x-demo` |

--- a/README.md
+++ b/README.md
@@ -49,9 +49,6 @@ recommend to set at least these values:
 | rabbitmq.rabbitmq.password           | Password for RabbitMq.                                                                     | `test`             |
 | global.postgresql.postgresqlPassword | Password for the Postgresql database.                                                      | `password`         |
 | global.redis.password                | Password for redis.                                                                        | `password`         |
-| securityContext.runAsUser            | The UID to run the entrypoint of container processes                                       | `1000`             |
-| securityContext.fsGroup              | A special supplemental group that applies to all containers in a pod                       | `1000`             |
-| securityContext.runAsNonRoot         | Indicates that the container must run as a non-root user                                   | `true`             |
 | Chart.appVersion                     | Version of Rasa X which you want to use.                                                   | `0.25.0`           |
 | rasa.tag                             | Version of Rasa OSS which you want to use.                                                 | `1.7.0`            |
 | app.name                             | Name of your action server image.                                                          | `rasa/rasa-x-demo` |

--- a/charts/rasa-x/Chart.yaml
+++ b/charts/rasa-x/Chart.yaml
@@ -1,7 +1,7 @@
 ---
 apiVersion: v1
 
-version: "1.0.4"
+version: "1.0.5"
 appVersion: "0.25.0"
 
 name: rasa-x

--- a/charts/rasa-x/values.yaml
+++ b/charts/rasa-x/values.yaml
@@ -279,8 +279,9 @@ images:
 
 # securityContext to use
 securityContext:
-#  # fsGroup to use (e.g. when deploying on Kubernetes)
-#  fsGroup: 0
+  runAsUser: 1000
+  fsGroup: 1000
+  runAsNonRoot: true
 
 # nameOverride replaces the Chart's name
 nameOverride: ""

--- a/charts/rasa-x/values.yaml
+++ b/charts/rasa-x/values.yaml
@@ -281,7 +281,6 @@ images:
 securityContext:
   runAsUser: 1000
   fsGroup: 1000
-  runAsNonRoot: true
 
 # nameOverride replaces the Chart's name
 nameOverride: ""

